fhir-parent/pom.xml

```diff
@@ -680,5 +679,193 @@
                 <com.ibm.fhir.model.spec.test.R4ExamplesDriver.testType>ALL</com.ibm.fhir.model.spec.test.R4ExamplesDriver.testType>
             </properties>
         </profile>
+        <profile>
+            <id>fhir-javadocs</id>
+            <build>
+                <plugins>
+                    <!-- limit the javadoc plugin to jenkins only -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.1.1</version>
+                        <configuration>
+                            <verbose>false</verbose>
+                            <doclint>none</doclint>
+                            <additionalparam>-Xdoclint:none</additionalparam>
+                            <minmemory>128m</minmemory>
+                            <maxmemory>1g</maxmemory>
+                            <archive>
+                                <manifestEntries>
+                                    <Automatic-Module-Name>com.ibm.fhir</Automatic-Module-Name>
+                                </manifestEntries>
+                            </archive>
+                            <destDir>target/apidocs/${project.artifactId}</destDir>
+                            <additionalOptions>
+                                <additionalOption>--frames</additionalOption>
+                            </additionalOptions>
+                            <bottom><![CDATA[Copyright 2019, <a href="http://www.ibm.com">IBM Corporation<a><br></br>FHIR&reg; is the registered trademark of HL7 and is used with the permission of HL7.]]></bottom>
+                            <author>false</author>
+                            <detectJavaApiLink>false</detectJavaApiLink>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>aggregate</id>
+                                <goals>
+                                    <goal>javadoc-no-fork</goal>
+                                    <goal>test-javadoc-no-fork</goal>
+                                </goals>
+                                <phase>site</phase>
+                                <configuration>
```

Comment:
why doesn't this inheret the default configuration?

Reply:
this the aggregation... slightly different purpose, and I found in this execution the configurations were not always picked up.